specification/compute/resource-manager/readme.md

```diff
@@ -238,7 +238,7 @@ input-file:
 - Microsoft.Compute/stable/2021-03-01/compute.json
 - Microsoft.Compute/stable/2021-03-01/runCommands.json
 - Microsoft.Compute/stable/2019-04-01/skus.json
-- Microsoft.Compute/stable/2020-12-01/disk.json
+- Microsoft.Compute/stable/2021-04-01/disk.json
```

Comment:
This is the first time 2021-04 is being introduced to Swagger, it cannot remain in the 2021-03-01 tag. Please make a new Tag for the 2021-04 release, and update the line at the top of this doc "tag: package-2021-03-01" to 2021-04.